### DISCUSSION TITLE
Export - File dimension?

### DISCUSSION THREAD

**Discussion by ctpntn** [kimai/kimai#2788] [Questions & Answers] [answered]
I need to export one year of data, but Kimai 2 1.10.2 stable (Ayumi) return me error. I have had to download blocks of three months.
I suppose is a dimension file problem. How can I resolve?

**Comment by kevinpapst** [comment, marked as answer]
You can try to increase the PHP setting `memory_limit` and max execution time

**Comment by ctpntn** [comment]
It worked! :)
